src/main/resources/META-INF/plugin.xml

```diff
@@ -78,6 +79,7 @@
 	<depends>org.jetbrains.plugins.terminal</depends>
 	<depends>org.jetbrains.idea.maven</depends>
 	<depends optional="true" config-file="javaee.xml">com.intellij.javaee</depends>
+	<depends optional="true" config-file="css.xml">com.intellij.css</depends>
```

Comment:
Does this 'optional=true' give us the ability to install with or without `com.intellij.css`?

Reply:
I assume so, so I'll go ahead and merge.